### PR TITLE
Added functions to print a message and symbol after stopping

### LINKED ISSUE #28: Function to add a symbol as well as text to the spinner
Currently, there are two functions, one for stopping the spinner with a symbol and one for stopping it with a text. However, it would be awesome if we could supply a custom symbol as well as custom text. Theoretically, I could supply the symbol along with the message but that means more code from the user's side.

We can also have functions like `succeed` and `fail` which will have a preset symbol like a green tick or red cross.

I will also try to make a pr for this

### PR DESCRIPTION
**Pull request by AnishDe12020**
Closes #28 

I have added a `stop_and_persist` function that stops the spinner and then shows a symbol and a message. I have also added `succeed`, `fail`, `warn`, `info`, and `unknown` functions that use the `stop_and_persist` function with a pre-defined symbol (with color and, theoretically, support for windows terminals).

Note: I might also push out some more features in this pr if it is going to take time (so that there arent many prs from me that sit unmerged). Do let me know if I should use any other approach

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -17,3 +17,4 @@ include = ["src/**/*", "README.md"]
 lazy_static = { version = "1.4.0" }
 maplit = { version = "1.0.2" }
 strum = { version = "0.24.0", features = ["derive"] }
+terminal-log-symbols = "0.1.6"
```
Comment (FGRibreau) [inline Cargo.toml:20]: I would prefer not to add another dependency, could we just keep your changes on stop_and_persist and stop_with_message? Thanks :)

Comment (AnishDe12020) [reply Cargo.toml:20]: Ok, I will revert the commits

Comment (AnishDe12020) [reply Cargo.toml]: Done

**Comment by FGRibreau** [comment]
Thanks a lot @AnishDe12020!

Released in v4.1.0 :)

**Pull request merged**